src/main/java/uk/gov/hmcts/reform/professionalapi/repository/ContactInformationRepository.java

```diff
@@ -1,13 +1,17 @@
 package uk.gov.hmcts.reform.professionalapi.repository;
 
 import java.util.UUID;
-
+import java.util.Set;
 import org.springframework.data.jpa.repository.JpaRepository;
-
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.stereotype.Repository;
 import uk.gov.hmcts.reform.professionalapi.domain.ContactInformation;
 
 @Repository
 public interface ContactInformationRepository extends JpaRepository<ContactInformation, UUID> {
 
+    @Modifying
+    @Query("Delete from contact_information ci where ci.id in (:idsSet)")
```

Comment:
do we really need this @Query? does not the jpa take care of itself if we use `deleteByAddressIdIn`

Reply:
there is no any such method in jpaRepository to delete.

Reply:
I tried with this `void deleteByIdIn(Set<UUID> idsSet);` and ran the functional test case which passed. Can you try the same and see if you are getting any issues?